docs/topics/proc_preparing-nodejs-application-for-openshift-deployment.adoc

```diff
@@ -0,0 +1,70 @@
+[id='preparing-a-application-for-openshift-deplyoment_{context}']
+= Preparing {Node} application for OpenShift deployment
+
+To prepare a {Node} application for OpenShift deployment, you must perform the following steps:
+
+* Add `nodeshift` to the application.
+* Add `openshift` and `start` entries to the `package.json` file.  
+
+.Prerequisites
+* `npm` installed.
+
+.Procedure
+. Add `nodeshift` to your application.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install nodeshift --save
+----
+
+. Add the `openshift` and `start` entries to the `scripts` section in `package.json`.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+{
+  "name": "myApp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "openshift": "nodeshift --strictSSL=false --metadata.out=deployment-metadata.json --build.forcePull=true {version-node-nodeshift}",
```

Comment:
you can probably get rid of most of those flag and it could be `"openshift":"nodeshift --expose {version-node-nodeshift}" `

I've added the `--expose` flag here which creates a route for you.  I mentioned in a previous comment that maybe we do the route thing here.

I would also like to get away from using that `.nodeshift` directory if possible